src/utils/api.js

```diff
@@ -22,6 +22,11 @@ export const getArticleById = async (article_id) => {
   return data;
 };
 
+export const patchArticleVotes = async (article_id, reqBody) => {
+  const { data } = await newsApi.patch(`/articles/${article_id}`, reqBody);
```

Comment:
With async/await, it's good practice to put the await function inside a try/catch block, so that you have an equivalent of a `.catch` block.